test/fuzz/SuiBridgeFuzzTest.t.sol

```diff
@@ -0,0 +1,135 @@
+        (committeeMemeberAddressD, committeeMemeberPkD) = makeAddrAndKey("D");
+        (committeeMemeberAddressE, committeeMemeberPkE) = makeAddrAndKey("E");
+
+        address[] memory _committeeMemebers = new address[](5);
+        _committeeMemebers[0] = committeeMemeberAddressA;
+        _committeeMemebers[1] = committeeMemeberAddressB;
+        _committeeMemebers[2] = committeeMemeberAddressC;
+        _committeeMemebers[3] = committeeMemeberAddressD;
+        _committeeMemebers[4] = committeeMemeberAddressE;
+
+        uint16[] memory _stake = new uint16[](5);
+        _stake[0] = committeeMemeberStakeA;
+        _stake[1] = committeeMemeberStakeB;
+        _stake[2] = committeeMemeberStakeC;
+        _stake[3] = committeeMemeberStakeD;
+        _stake[4] = committeeMemeberStakeE;
+
+        bridgeCommittee.initialize(_committeeMemebers, _stake);
+
+        bridgeLimiter = new BridgeLimiter();
+
+        uint256 totalLimit = 10000000000;
+
+        address[] memory _supportedTokens = new address[](4);
+        _supportedTokens[0] = wBTC;
+        _supportedTokens[1] = wETH;
+        _supportedTokens[2] = USDC;
+        _supportedTokens[3] = USDT;
+
+        bridgeTokens = new BridgeTokens(_supportedTokens);
+
+        uint256[] memory assetPrices = new uint256[](4);
+        assetPrices[0] = SUI_PRICE;
+        assetPrices[1] = BTC_PRICE;
+        assetPrices[2] = ETH_PRICE;
+        assetPrices[3] = USDC_PRICE;
+
+        bridgeLimiter.initialize(
+            address(bridgeCommittee),
+            address(bridgeTokens),
+            assetPrices,
+            totalLimit
+        );
+
+        suiBridge = new SuiBridge();
+        suiBridge.initialize(
+            address(bridgeCommittee),
+            address(bridgeTokens),
+            address(bridgeVault),
+            address(bridgeLimiter),
+            wETH,
+            chainID
+        );
+    }
+
+    function testFuzz_executeEmergencyOpWithSignatures(
+        uint8 isFreezing
+    ) public {
+        isFreezing = uint8(bound(isFreezing, 0, 1));
+        // Create emergency op message
+        BridgeMessage.Message memory message = BridgeMessage.Message({
+            messageType: BridgeMessage.EMERGENCY_OP,
+            version: 1,
+            nonce: 0,
+            chainID: 1,
+            payload: abi.encode(isFreezing)
+        });
+
+        bytes memory encodedMessage = BridgeMessage.encodeMessage(message);
+        bytes32 messageHash = keccak256(encodedMessage);
+
+        bytes[] memory signatures = new bytes[](5);
+        signatures[0] = getSignature(messageHash, committeeMemeberPkA);
+        signatures[1] = getSignature(messageHash, committeeMemeberPkB);
+        signatures[2] = getSignature(messageHash, committeeMemeberPkC);
+        signatures[3] = getSignature(messageHash, committeeMemeberPkD);
+        signatures[4] = getSignature(messageHash, committeeMemeberPkE);
+
+        // suiBridge.executeEmergencyOpWithSignatures(signatures, message);
+    }
+}
```

Comment:
We'll want fuzz tests for the `BridgeMessage` library too (mostly the decoding functions). Do this last tho because we're still waiting on the decoding to be pinned down for some of the decoding functions.